docs/data/material/components/autocomplete/ComboBox.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import TextField from '@mui/material/TextField';
 import Autocomplete from '@mui/material/Autocomplete';
+import top100Films from './data/top100Films.js';
```

Comment:
Do you mean that the relative imported package should always be a `.ts` file? 

What I'm thinking is if there is no extension, it would become a lot more complicated to figure out which file to load when resolving these relative imports in `loader.js`

Reply:
@alexfauquette By as it is do you mean keep the `.js` extension?

Reply:
yes